### PR TITLE
chore(blueprint): 💅 Tag and Lozenge - New Design System

### PR DESCRIPTION
**Pull request by diegographext**
Left:
- [x] Review styles with design system

[JIRA](https://graphext.atlassian.net/browse/GRAPH-7318)
[Graphext Storybook PR](https://github.com/graphext/graphext/pull/2108)
[Preview of Storybook with blueprint styles for TAG](https://g-storybook-tag.netlify.app/?path=/docs/design-system-components-tag--default)
[Preview of Storybook with blueprint styles for LOZENGE](https://g-storybook-tag.netlify.app/?path=/docs/design-system-components-newlozenge--default)

### CODE REVIEW

File: packages/core/src/common/_customs.scss [hunk 1]
```diff
@@ -60,7 +61,8 @@ $red4: #f55656;
 $red5: #ff7373;
 
 $white1: #ffffff;
-$white2: rgba(255, 255, 255, 0.8);
+$white2: rgba(255, 255, 255, 0.8
+);
```
Comment (crispamares) [inline packages/core/src/common/_customs.scss]:
```suggestion
$white2: rgba(255, 255, 255, 0.8);
```

File: packages/core/src/components/tag/_overrides.scss [hunk 2]
```diff
@@ -1,36 +1,261 @@
...
+    @include custom-tag-kind(insight, $g-wizard-26-700, $g-wizard-26-200);
+    @include custom-tag-kind(publish, $g-water-700, $g-water-200);
+    @include custom-tag-kind(project, $g-wizard-18-700, $g-wizard-18-200);
+    @include custom-tag-kind(danger, $g-error-700, $g-error-200);
+    @include custom-tag-kind(updated, $g-grass-700, $g-grass-200);
+    @include custom-tag-kind('active', $g-avocado-700, $g-avocado-200);
+    @include custom-tag-kind(inRecipe, $g-wizard-4-700, $g-wizard-4-200);
+    @include custom-tag-kind(notInRecipe, $g-wood-700, $g-wood-200);
+    @include custom-tag-kind(recreate, $g-water-700, $g-water-200);
+    @include custom-tag-kind(pending, $g-grey1, $g-base2);
```
Comment (crispamares) [inline packages/core/src/components/tag/_overrides.scss]: I suggest to change the `kind` by `color` and use the name of the color for the values. Check the design system, it is updated with new colors (no use of colors named `wizard_xxx`)

**Review by crispamares — CHANGES_REQUESTED** [review]

File: packages/core/src/common/_customs.scss [hunk 3]
```diff
@@ -338,4 +340,52 @@ $g-wizard-10-500: #517F58;
...
+$g-lavander-50: #FAF8FC;
+$g-lavander-100: #F2EDF7;
+$g-lavander-200: #E8DFF1;
+$g-lavander-300: #D4C3E5;
+$g-lavander-400: #C3AADA;
+$g-lavander-500: #AA86CB;
+$g-lavander-600: #9467BD;
+$g-lavander-700: #60437A;
+$g-lavander-800: #463159;
+$g-lavander-900: #2C1F38;
```
Comment (LunaFidalgo) [inline packages/core/src/common/_customs.scss]: I think we have a typo here... it should be "lavEnder" 😢 

image

File: packages/core/src/components/tag/_overrides.scss (hunk 2 above)
Comment (diegographext) [reply packages/core/src/components/tag/_overrides.scss]: Updated!

**Pull request merged**